app/invocation/cache_requests_card.tsx

```diff
@@ -371,26 +371,26 @@ export default class CacheRequestsCardComponent extends React.Component<CacheReq
   private renderResultHovercard(result: cache.ScoreCard.IResult, startTimeMillis: number) {
     return (
       <div className="cache-result-hovercard">
-        {result.targetId && (
+        {result.targetId ? (
           <>
             <b>Target</b> <span>{result.targetId}</span>
           </>
-        )}
-        {(result.actionMnemonic || result.actionId) && (
+        ) : null}
+        {result.actionMnemonic || result.actionId ? (
```

Comment:
is removing the `()` intentional here?

Reply:
Yeah the buildfix script removed it. I think it's implied now that the && is removed and order of operations is no longer important